routemaster/client/assertion_helpers.rb

```diff
@@ -0,0 +1,18 @@
+require 'uri'
+
+module Routemaster
+  class Client
```

Comment:
You're reopening a class, which can easily cause load order issues.
If you change it to be a modules (~ singleton) this goes away though.